Mathlib/Topology/Order/UpperLowerSetTopology.lean

```diff
@@ -308,8 +308,9 @@ instance _root_.OrderDual.instIsUpperSet [Preorder α] [TopologicalSpace α] [To
   topology_eq_upperSetTopology := by ext; rw [IsLowerSet.topology_eq α]
 
 /-- If `α` is equipped with the lower set topology, then it is homeomorphic to `WithLowerSet α`. -/
-def WithLowerSetHomeomorph : WithLowerSet α ≃ₜ α :=
-  WithLowerSet.ofLowerSet.toHomeomorphOfIsInducing ⟨by erw [topology_eq α, induced_id]; rfl⟩
+def WithLowerSetHomeomorph : WithLowerSet α ≃ₜ α := by
```

Comment:
This adds `cast` to **data** part of the construction. Please keep `rw`s in proofs.

Reply:
Oops, missed one, thanks.